tests/spec/core/biblio-spec.js

```diff
@@ -92,26 +117,104 @@ describe("W3C — Bibliographic References", () => {
     expect(ref.textContent).toMatch(/Publishers Inc\.\s/);
     ref = null;
     // Make sure the ". " is automatically added to publisher.
-    ref = doc.querySelector("#bib-TestRef2 + dd");
+    ref = docDefault.querySelector("#bib-TestRef2 + dd");
     expect(ref).toBeTruthy();
     expect(ref.textContent).toMatch(/Testing 123\.\s/);
     ref = null;
     // Make sure publisher is shown even when there is no author
-    ref = doc.querySelector("#bib-TestRef3 + dd");
+    ref = docDefault.querySelector("#bib-TestRef3 + dd");
     expect(ref).toBeTruthy();
     expect(ref.textContent).toMatch(/Publisher Here\.\s/);
   });
 
+  it("displays the reference for APA citation", () => {
+    // Make sure the reference is added.
+    let ref = docAPA.querySelector("#bib-TestRef1 + dd");
+    expect(ref).toBeTruthy();
+    expect(ref.textContent).toMatch(/William Shakespeare\.\s/);
+    expect(ref.textContent).toMatch(/Test ref title\.\s/);
+    expect(ref.textContent).toMatch("Retrieved from URL: http://test.com/");
+    ref = null;
```

Comment:
If you find you need to reset things, then this signals you need a new test, so here:
```JS
it(".... describe what is different...", ()=>{

}); 
```